src/core/tests/test_schema.py

```diff
@@ -0,0 +1,51 @@
+import schemathesis
+from hypothesis import settings, HealthCheck
+
+schema = schemathesis.from_uri("http://127.0.0.1:5000/api/v1/doc/swagger.json")
```

Comment:
FWIW, it should load YAML files too, if it doesn't, please, consider opening [a bug report ](https://github.com/schemathesis/schemathesis/issues/new?assignees=Stranger6667&labels=Status%3A+Review+Needed%2C+Type%3A+Bug&projects=&template=bug_report.md&title=%5BBUG%5D)so it could be fixed on the Schemathesis side

Reply:
solved with `from_wsgi` and `from_pytest_fixture`